po/fr.po

```diff
@@ -131,139 +137,129 @@ msgstr "Snippet Pixie"
 
 #: src/MainWindow.vala:120
 msgid "Update Existing"
-msgstr ""
+msgstr "Mettre à jour l'existant"
 
 #: src/MainWindow.vala:121
 msgid "Cancel"
-msgstr ""
+msgstr "Annuler"
 
 #: src/MainWindow.vala:122
 msgid "Skip Duplicates"
-msgstr ""
+msgstr "Ignorer les doublons"
 
 #: src/MainWindow.vala:149
 msgid "Imported Snippets"
-msgstr ""
+msgstr "Raccourcis importés"
 
 #: src/MainWindow.vala:149
 msgid "Your snippets were successfully imported."
-msgstr ""
+msgstr "Vos raccourcis ont été importés avec succès."
 
 #: src/MainWindow.vala:153
 msgid "Failed to import selected file"
-msgstr ""
+msgstr "Échec de l'importation du fichier sélectionné"
 
 #: src/MainWindow.vala:153
 msgid ""
 "Snippet Pixie can currently only import the JSON format files that it also "
 "exports."
-msgstr ""
+msgstr "Snippet Pixie ne peut actuellement importer que les fichiers au format JSON qu'il exporte "
+"également."
 
 #: src/MainWindow.vala:161
 msgid "Export Snippets"
-msgstr ""
+msgstr "Exporter les raccourcis"
 
 #: src/MainWindow.vala:169
 msgid "Exported Snippets"
-msgstr ""
+msgstr "Raccourcis exportés"
 
 #: src/MainWindow.vala:169
 msgid "Your snippets were successfully exported."
-msgstr ""
+msgstr "Vos raccourcis ont été exportés avec succès."
 
 #: src/MainWindow.vala:173
 msgid "Failed to export to file"
-msgstr ""
+msgstr "Échec de l'exportation du fichier"
 
 #: src/MainWindow.vala:173
 msgid "Something went wrong, sorry."
-msgstr ""
+msgstr "Une erreur s'est produite, désolé."
 
 #: src/MainWindowHeader.vala:26
 msgid "Add snippet"
-msgstr ""
+msgstr "Ajouter un raccourci"
 
 #: src/MainWindowHeader.vala:44
 msgid "Import snippets…"
-msgstr ""
+msgstr "Importer des raccourcis…"
 
 #: src/MainWindowHeader.vala:46
 msgid "Export snippets…"
-msgstr ""
+msgstr "Exporter des raccourcis…"
 
 #: src/MainWindowHeader.vala:48
 msgid "About…"
-msgstr ""
+msgstr "À propos…"
 
 #: src/Snippet.vala:22
 msgid "new"
-msgstr ""
+msgstr "Nouveau"
 
 #: src/Snippet.vala:23
 msgid "Something to be replaced"
-msgstr ""
+msgstr "Quelque chose à remplacer"
 
 #: src/ViewStack.vala:55
 msgid "Abbreviation"
-msgstr ""
+msgstr "Abbréviation"
 
 #: src/ViewStack.vala:64
 msgid "Body"
-msgstr ""
+msgstr "Corps"
 
 #: src/WelcomeView.vala:22
 msgid "No snippets found."
-msgstr ""
-
-#: src/WelcomeView.vala:23
-msgid "document-new"
-msgstr ""
+msgstr "Aucun raccourci trouvé."
 
 #: src/WelcomeView.vala:23
 msgid "Add Snippet"
-msgstr ""
+msgstr "Ajouter un raccourci"
 
 #: src/WelcomeView.vala:23
 msgid "Create your first snippet."
-msgstr ""
-
-#: src/WelcomeView.vala:24
-msgid "document-import"
-msgstr ""
+msgstr "Créez votre premier raccourci."
 
 #: src/WelcomeView.vala:24
 msgid "Import previously exported snippets."
-msgstr ""
-
-#: src/WelcomeView.vala:25
-msgid "help-contents"
-msgstr ""
+msgstr "Importez vos raccourcis précédemment exportés."
 
 #: src/WelcomeView.vala:25
 msgid "Quick Start Guide"
-msgstr ""
+msgstr "Guide de démarrage rapide
```

Comment:
Fails Travis CI build here due to missing `"`.

Reply:
Just fixed